app/controllers/api/v1/search_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class SearchController < Api::V1::BaseApiController
+      include SearchDoc
+
+      skip_before_filter :authenticate_from_token!
+
+      def create
+        query_string = params[:query]
+        repository = Elasticsearch::Persistence::Repository.new
+        repository.index = 'checkdesk_application_development_annotations'
```

Comment:
The index name should not be hard-coded... try to read this property: https://github.com/meedan/check-api/blob/develop/app/models/concerns/annotation_base.rb#L66